packages/ovh-manager-core/session/session.service.js

```diff
@@ -0,0 +1,31 @@
+export default class SessionService {
+  constructor($q, OvhApiMe) {
+    'ngInject';
+
+    this.$q = $q;
+    this.OvhApiMe = OvhApiMe;
+  }
+
+  getUser(force) {
+    // Check if cached
+    if (this.user) {
+      return this.$q.when(this.user);
```

Comment:
`user` should be private to avoid access without "promise"

`getUser` returns a promise
`user` return an object

You should use a scoped variable :)